CMakeLists.txt

```diff
@@ -860,7 +881,73 @@ if(NOT DEFINED SWIFT_API_NOTES_INPUTS)
   message(FATAL_ERROR "API notes are not available in ${SWIFT_API_NOTES_PATH}")
 endif()
 
+#
+# Configure Swift to build with profile generation
+#
+
+if(SWIFT_ENABLE_PROFILE_GENERATION OR SWIFT_ENABLE_IR_PROFILE_GENERATION)
+  if(NOT SWIFT_PROFILE_MERGE_POOL_SIZE)
+    # A pool size of 1-2 is probably sufficient on a SSD. 3-4 should be fine
+    # for spining disks. Anything higher may only help on slower mediums.
+    set(SWIFT_PROFILE_MERGE_POOL_SIZE "4")
```

Comment:
Do you have any concrete numbers that describe how the size of the merge pool affects build times?